### PR TITLE
TW-935 v10.10 back/forward buttons

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

**Review by HeatherStenson — APPROVED** [review]
Looks great! ✨ 🍊

File: src/pages/docs/getting-started/navigating-postman.md [hunk 1]
```diff
@@ -53,14 +53,15 @@ Postman has a variety of tools, views, and controls to help you manage your API
...
+<img alt="Postman app" src="https://assets.postman.com/postman-docs/postman-desktop-app-v10.10.jpg"/>
 
 ## Header
 
 The header enables you to create workspaces, access reports, explore the public API network, search in Postman, view sync status and notifications, and access your settings, account, and [Postman plan](/docs/administration/buying/).
 
-<img alt="Postman header left side" src="https://assets.postman.com/postman-docs/navigating-postman-header-left-v9.19.jpg" width="350px"/>
+<img alt="Postman header left side" src="https://assets.postman.com/postman-docs/postman-desktop-app-header-v10.10.jpg" width="350px"/>
 
+* **&#8592; &#8594;** - Navigate backwards and forwards through pages you've visited within Postman. _(Postman desktop app only)_
```
Comment (petewitcher-postman) [inline src/pages/docs/getting-started/navigating-postman.md:64]:
```suggestion
* **&#8592; &#8594;** - Navigate backward and forward through pages you've visited within Postman. _(Postman desktop app only)_
```

**Review by petewitcher-postman — APPROVED** [review]
One little suggestion, LGTM!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**